### PR TITLE
Update proxy-installs.md - Make clear the call to allow the script use ENV VARs values

### PR DESCRIPTION
**Pull request by camilamacedo86**
…e ENV VARs values

### CODE REVIEW

File: src/markdown-pages/install-with-kurl/proxy-installs.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ If a `proxyAddress` is not configured in the installer spec, the following envir
 
 Any addresses set in either the `NO_PROXY` or `no_proxy` environment variable will be added to the list of no proxy addresses.
 
+_**WARNIN**_: You must to call the script with -E in order to allow the script use the environment variables (i.e. `curl https://kurl.sh/latest | sudo -E bash` )
```
Comment (rrpolanco) [inline src/markdown-pages/install-with-kurl/proxy-installs.md:31]:
```suggestion
_**WARNING**_: You must call the script with -E in order to allow the script to use the environment variables (i.e. `curl https://kurl.sh/latest | sudo -E bash` )
```

**Review by emosbaugh — COMMENTED** [review]
I think it is more common in a real customer environment for someone to have a proxy set on the server rather than exporting it to the current environment. I am not sure this clarification is needed.

**Comment by camilamacedo86** [comment]
>I think it is more common in a real customer environment for someone to have a proxy set on the server rather than exporting it to the current environment. I am not sure this clarification is needed.

How an user will know by reading the doc that they need to use -E in when the proxy is set via ENV VAR?

**Comment by emosbaugh** [comment]
> >I think it is more common in a real customer environment for someone to have a proxy set on the server rather than exporting it to the current environment. I am not sure this clarification is needed.
> 
> 
> 
> How an user will know by reading the doc that they need to use -E in when the proxy is set via ENV VAR?
> 
> 

They would only need to set that if it's set in their current environment only.

**Comment by camilamacedo86** [comment]
HI @emosbaugh,

>They would only need to set that if it's set in their current environment only.

Yes, that is fine. However, in the doc we say that we use the value for their environment but we do NOT say that for that be possible they need to run the script with -E

**Comment by emosbaugh** [comment]
> HI @emosbaugh,
> 
> > They would only need to set that if it's set in their current environment only.
> 
> Yes, that is fine. However, in the doc we say that we use the value for their environment but we do NOT say that for that be possible they need to run the script with -E

But -E is not a feature of kurl. That is a feature of the linux shell.

**Pull request closed without merging**